batch7crud-roll8/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,260 @@
+/* Sign In Page*/
+
+.signin-form {
+  width: 40%;
+  margin: 50px auto;
+  background-color: #F39C12;
+}
+
+.signin-form-content {
+  padding: 70px;
+}
+
+.signin-form-content p {
+  padding: 5px;
+  text-align: center;
+}
+
+.signin-inputs {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Check this potential box model size issue [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Abox-model)